wallet_test.go

```diff
@@ -141,3 +144,65 @@ func TestWalletSignAndRecoverConfigOfMultipleSigners(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, wallet.Address(), address)
 }
+
+func TestShouldNotSideEffectWalletConfig(t *testing.T) {
+	eoa1, err := ethwallet.NewWalletFromRandomEntropy()
+	assert.NoError(t, err)
+
+	eoa2, err := ethwallet.NewWalletFromRandomEntropy()
+	assert.NoError(t, err)
+
+	// Inverse-sort signers
+	signers := sequence.WalletConfigSigners{
+		{Weight: 2, Address: eoa1.Address()},
+		{Weight: 5, Address: eoa2.Address()},
+	}
+
+	sort.Sort(sort.Reverse(signers))
```

Comment:
Is this correct? Seems weird to reverse the signers, then sort. Maybe it should be sort first, then reverse?

Reply:
Never mind, this is correct and I'm wrong here.